lms/djangoapps/certificates/views.py

```diff
@@ -461,18 +462,24 @@ def render_html_view(request, user_id, course_id):
 
     # Create the initial view context, bootstrapping with Django settings and passed-in values
     context = {}
-    context['platform_name'] = settings.PLATFORM_NAME
+    context['platform_name'] = microsite.get_value("platform_name", settings.PLATFORM_NAME)
     context['course_id'] = course_id
 
     # Update the view context with the default ConfigurationModel settings
     configuration = CertificateHtmlViewConfiguration.get_config()
-    context.update(configuration.get('default', {}))
+    # if we are in a microsite, then let's first see if there is an override
+    # section in our config
+    config_key = microsite.get_value('microsite_config_key', 'default')
+    # if there is no special microsite override, then let's use default
+    if config_key not in configuration:
+        config_key = 'default'
+    context.update(configuration.get(config_key, {}))
 
```

Comment:
Hrm, OK, I see what you mean. The trouble is that we need to make sure that microsite defined overrides "win" and that they are not overwritten by the call to _update_certificate_context() down below. We may have to reapply the microsite sourced config after the call to _update_certificate_context().

Actually, I have an idea, let me make a branch of this branch and you/matt can review it.